src/main.rs

```diff
@@ -1,3 +1,32 @@
-fn main() {
-    println!("Hello, world!");
+use crate::collectors::register_collectors;
```

Comment:
Shouldn't this be `use monitorbot::collectors::register_collectors;`?

Reply:
yes. in fact, that import should be completely removed, and the `fn` I think should be brought into scope in line 3 since we already importing `MetricsProvider` from `monitorbot`. ex: 
```rust
use monitorbot::{MetricProvider, collectors::register_collectors};
```
👍